content/event/002.open-tech-meetup-2024-08.en.md

```diff
@@ -10,9 +10,8 @@ end_time: 2024-08-06T23:00:00+02:00
 location: Andreas-Pfitzmann-Bau (APB) Room E023 
 ---
 
-A lot has happened and we want to tell you about it. The most important thing is that the DD-IX is now running 
-productively at two locations. But there have also been some developments in the community that are worth 
-reporting. We are also looking forward to another very productive exchange of ideas and some fun.
+A lot has happened since our last Tech Meetup two months ago. First things first: we've moved into the CityCentre and 
+our infrastructure is up and running! We would like to talk to you about this and the latest news from the DD-IX community.
```

Comment:
... we would like to share more details and latest news from the DD-IX community with you.